proto/control.proto

```diff
@@ -11,12 +11,15 @@ service PDPControl {
 message Item {
   enum DataType {
     POLICIES = 0;
-    CONTENT  = 1;
+    CONTENT = 1;
+    POLICIES_PATCH = 2;
+    CONTENT_PATCH = 3;
   }
-           DataType type     = 1;
-           int32    dataId   = 2;
-           string   id       = 3;
-  repeated int32    includes = 4;
+  DataType type = 1;
+  string version = 2;
```

Comment:
This is actually ```to witch version```. It's assumed that PAP tracks current version for each PDP and creates patch from this version to latest one and sends only latest one. In order to validate on PDP side it's perhaps make sense to send both version - from and to...